src/App.jsx

```diff
@@ -8,7 +8,7 @@ import ArticlePage from "./components/ArticlePage";
 import "./App.css";
 
 function App() {
-  const [currentUser, SetCurrentUser] = useState("");
+  const [currentUser, SetCurrentUser] = useState("jessjelly");
```

Comment:
setCurrentUser should be camelCase syntax - good idea to hard code the user!

Reply:
OK- updated